docs/how-tos/image-data.md

```diff
@@ -120,8 +127,8 @@ Add an ML model service on your machine that is compatible with the ML model you
 {{<imgproc src="/services/ml/train.svg" class="fill alignleft" style="max-width: 150px"  declaredimensions=true alt="Train models">}}
 **2. Select a suitable ML model**
 
-From the **Model** dropdown on the ML model service configuration panel, select an [existing model](https://app.viam.com/registry?type=ML+Model) you want to use, or click **Add new model** to upload your own.
-If you're not sure which model to add, you can add [`EfficientNet-ImageNet2012`](https://app.viam.com/ml-model/viam/EfficientNet-ImageNet2012) which can detect people and animals, among other things.
+Click **Select model** on the ML model service configuration panel, then select an [existing model](https://app.viam.com/registry?type=ML+Model) you want to use, or click **Add new model** to upload your own.
+If you're not sure which model to add, you can add [`face-detection`](https://app.viam.com/ml-model/bijan/face-detection) from the **Registry**, which can detect if a person's face appears in the image stream.
```

Comment:
because you just use add as "adding a model by uploading it"
```suggestion
If you're not sure which model to add, you can use [`face-detection`](https://app.viam.com/ml-model/bijan/face-detection) from the **Registry**, which can detect if a person's face appears in the image stream.
```